warehouse/integrations/testhelper/setup.go

```diff
@@ -560,14 +560,37 @@ func verifyAsyncJob(t testing.TB, wareHouseTest *WareHouseTest) {
 		operation,
 		WaitFor10Minute,
 		AsyncJOBQueryFrequency,
-		fmt.Sprintf("Failed to get async job status for job_run_id: %s, task_run_id: %s, source_id: %s, destination_id: %s",
+		fmt.Sprintf("Failed to get async job status for job_run_id: %s, task_run_id: %s, source_id: %s, destination_id: %s, woekspace_id: %s",
 			wareHouseTest.JobRunID,
 			wareHouseTest.TaskRunID,
 			wareHouseTest.SourceID,
 			wareHouseTest.DestinationID,
+			wareHouseTest.WorkspaceID,
 		),
 	)
 
+	sqlStatement := `
+	SELECT status, workspace_id
+	FROM wh_async_jobs 
+	WHERE source_id=$1
+	AND destination_id=$2
+	AND workspace_id=$3
+	AND metadata->>'job_run_id'=$4 
+	AND metadata->>'task_run_id'=$5; 
+	`
```

Comment:
We can probably format the query. wdty?
```suggestion
	sqlStatement := `
		SELECT 
		  status, 
		  workspace_id 
		FROM 
		  wh_async_jobs 
		WHERE 
		  source_id = $1 AND 
		  destination_id = $2 AND
		  workspace_id = $3 AND
		  metadata ->> 'job_run_id' = $4 AND
		  metadata ->> 'task_run_id' = $5;
	`
```